proxy/hdrs/MIME.cc

```diff
@@ -2570,8 +2570,12 @@ mime_parser_parse(MIMEParser *parser, HdrHeap *heap, MIMEHdrImpl *mh, const char
     // server MUST reject any received request message that contains
     // whitespace between a header field-name and colon with a response code
     // of 400 (Bad Request).
+    //A proxy MUST remove any such whitespace from a response message before
+    // fowarding the message downstream.
     if (is_ws(field_name.back())) {
-      return PARSE_RESULT_ERROR;
+	  if (!remove_ws_from_field_name)
```

Comment:
Please add braces for this if statement.

Reply:
OK!~